Meta/Lagom/CMakeLists.txt

```diff
@@ -191,6 +191,17 @@ if (BUILD_LAGOM)
         target_link_libraries(sql_lagom Lagom)
         target_link_libraries(sql_lagom stdc++)
 
+        add_executable(test-iodevice ../../Tests/LibCore/TestLibCoreIODevice ${LIBTEST_MAIN})
+        set_target_properties(test-iodevice PROPERTIES OUTPUT_NAME test-iodevice)
+        target_link_libraries(test-iodevice Lagom)
+        target_link_libraries(test-iodevice LagomTest)
+        target_link_libraries(test-iodevice stdc++)
+        add_test(
+            NAME test-iodevice
+            COMMAND test-iodevice
+            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../../Tests/LibCore
+        )
+
```

Comment:
I don't think we need this in Lagom. Does it even work? The `add_executable()` path seems to be missing a `.cpp` suffix.

Reply:
This works, if for example I change the filename I get a failure:
      Start  4: test-iodevice
 4/89 Test  #4: test-iodevice ........................Child aborted***Exception:   0.17 sec

It doesn't need the cpp extension. 

Also I was following the suggestion of alimpfard to add it to Lagom.

Reply:
Fine, but please add the cpp extension, like we do for literally every other `add_executable()` here as well.

Reply:
Done